services/node_create.proto

```diff
@@ -27,62 +25,90 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * A transaction body to add a new node to the network.
- * After the node is created, the node_id for it is in the receipt.
+ * A transaction body to add a new consensus node to the network address book.
  *
  * This transaction body SHALL be considered a "privileged transaction".
  *
- * This message supports a transaction to create a new node in the network.
- * The transaction, once complete, enables a new consensus node
+ * This message supports a transaction to create a new node in the network
+ * address book. The transaction, once complete, enables a new consensus node
  * to join the network, and requires governing council authorization.
  *
- * A `NodeCreateTransactionBody` MUST be signed by the governing council.<br/>
- * The newly created node information will be used to generate config.txt and
- * a-pulbic-NodeAlias.pem file per each node during phase 2,<br>
- * not active until next freeze upgrade.
+ * - A `NodeCreateTransactionBody` MUST be signed by the governing council.
+ * - The newly created node information SHALL be added to the network address
+ *   book information in the network state.
+ * - The new entry SHALL be created in a "pending" state.
```

Comment:
We should remove "pending" state here.

Reply:
What state is the internal (in the merkle tree) entry created, if not "pending"?  I just copied that from the previous text and reworded.

Reply:
I've updated this text with a possible reword; it's a bit of a guess, so please review carefully.